src/data-pipeline/utils/s3-asset.ts

```diff
@@ -36,7 +36,7 @@ export function uploadBuiltInSparkJarsAndFiles(
 
   let bundling: BundlingOptions = {
     user: 'gradle',
-    image: DockerImage.fromRegistry('public.ecr.aws/docker/library/gradle:7.6-jdk11'),
+    image: DockerImage.fromRegistry('public.ecr.aws/docker/library/gradle:7.6-jdk17'),
```

Comment:
why not bumping to latest gradle?

Reply:
gradle 8 has breaking change.  the jacocoTestReport does not work.